src/Integration.Vsix/Commands/HelpMenu/ReportProblemCommand.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using SonarLint.VisualStudio.IssueVisualization.Helpers;
+
+namespace SonarLint.VisualStudio.Integration.Vsix.Commands.HelpMenu
+{
+    internal class ReportProblemCommand : VsCommandBase
+    {
+        internal const int Id = 0x105;
+
+        private readonly IShowInBrowserService browserService;
+
+        public ReportProblemCommand(IShowInBrowserService browserService)
+        {
+            this.browserService = browserService;
+        }
+
+        protected override void InvokeInternal()
+        {
+            browserService.ShowProblemReportPage();
```

Comment:
Since this will also be used for feedback, maybe we should call ShowCommunityPage

And perhaps the same for the command 🤔 wdyt? the label "report a problem" might chance